ui/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "9volt-ui",
+  "version": "1.0.0",
+  "description": "User Interface for interacting with 9volt",
+  "main": "index.jsx",
+  "scripts": {
+    "dev": "webpack -d --watch --disable-nodeenv-plugin --disable-uglify-plugin",
+    "build": "webpack -p --bail"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/9corp/9volt.git"
+  },
+  "author": "9corp",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/9corp/9volt/issues"
+  },
+  "homepage": "https://github.com/9corp/9volt#readme",
+  "dependencies": {
```

Comment:
The `package.json` is very much the project dependencies management file. Everything is really in here and almost everything you see here is managed by the `npm` command and not by hand. `dependencies` is what the runtime needs to have to run (think `vendor` in go) and `devDependencies` is what packages are required for build. Everything is here for CI/CD and should easily facilitate building the UI on TravisCI.